data/documents_big.xml

```diff
@@ -12397,7 +12397,7 @@
     <subfield code="2">rdacarrier</subfield>
   </datafield>
   <datafield tag="490" ind1="1" ind2=" ">
-    <subfield code="a">Texte intégral</subfield>
+    <subfield code="a">fullText</subfield>
```

Comment:
fulltext or Texte intégral? Present at several places in this document.

Reply:
In virtua it is free text, but in rero-ils we need keywords.